src/compositions/templates/case-study/CustomerQuote/styled.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { highlight, textLighter, textOnDark } from 'styles/colours';
+
+export const QuoteWrapper = styled.div`
+  color: ${textOnDark};
+  margin: -13% 7% 2% 7%;
+  background-color: ${highlight};
+  position: relative;
+  font-weight: 300;
+  line-height: 1.24;
+`;
+
+export const Container = styled.div`
+  background-color: transparent;
+`;
+
+export const Quote = styled.p`
+  font-size: 2.375rem;
+  padding: 77px 111px 20px 107px;
```

Comment:
Remove padding from here, and add it to the container instead.
Just have a margin-bottom on the paragraph element